fem/fespace.hpp

```diff
@@ -169,31 +187,80 @@ class FiniteElementSpace
        boundary. */
    void BuildNURBSFaceToDofTable() const;
 
+   typedef std::uint64_t VarOrderBits;
+   static constexpr int MaxVarOrder = 8*sizeof(VarOrderBits) - 1;
```

Comment:
A description of how this data type is used would be nice.

Reply:
Added.